jwst/associations/lib/dms_base.py

```diff
@@ -19,6 +19,21 @@
 _ASN_NAME_TEMPLATE_STAMP = 'jw{program}-{acid}_{stamp}_{type}_{sequence:03d}_asn'
 _ASN_NAME_TEMPLATE = 'jw{program}-{acid}_{type}_{sequence:03d}_asn'
 
+# Acquistions and Confirmation images
+ACQ_EXP_TYPES = (
+    'mir_tacq',
+    'nis_taconfirm',
+    'nis_tacq',
+    'nrc_taconfirm',
+    'nrc_tacq',
+    'nrs_confirm',
+    'nrs_msata',
+    'nrs_taconfirm',
+    'nrs_tacq',
+    'nrs_taslit',
+    'nrs_wata',
+)
```

Comment:
Let's hope that's a complete and accurate list. Things seem to be fairly fluid yet.

Reply:
Hope so too.  For this PR, I'm using just PNTGTYPE to determine the TSO-ness of an exposure without using this list. It would be worth a refactor whatever rules use this list to just depend on PNTGTYPE.